components/istio-kyma-patch/README.md

```diff
@@ -2,38 +2,35 @@
-Istio's CRDs which are required by kyma. 
+## Usage
+
+The application performs several steps:
+1. Check if the required CRDs are already deployed. The script reads the `required-crds` file which should contain a 
+list of Istio's CRDs that Kyma requires. If CRDs are not deployed patch will fail.
 
-1. The configuration of sidecar injector. The following changes are introduced to the `istio-sidecar-injector` ConfigMap:
+2. Configure the sidecar injector. The following changes are introduced to the `istio-sidecar-injector` ConfigMap:
     * The **policy** parameter is set to `disabled`.
-    * The **zipkinAddress** points to zipkin deployed in `kyma-system` Namespace.
-    * All containers have set default **resources.limits.memory** to `128Mi` and **resources.limits.cpu** to `100m`.
+    * The **zipkinAddress** points to Zipkin deployed in the `kyma-system` Namespace.
+    * All containers have the default **resources.limits.memory** set to `128Mi` and **resources.limits.cpu** to `100m`.
 
-1. Istio components are patched. The script is expecting files in format `<resource-name>.<kind>.patch.json` which contain
-patch in JsonPatch format. Components are patched using `kubectl patch` command. Patched resource may not exist. 
-If other failure occurs the script must fail. See [job ConfigMap](../../resources/istio-kyma-patch/templates/configmap.yaml) 
-to see which patches are applied by default.
+3. Patch Istio components. The script looks for files in the `{resource-name}.{kind}.patch.json` format which contain a 
+`JsonPatch`. The components are applied using the `kubectl patch` command. The modified resource may not exist. Patch 
+will be skipped for such resource. On any other failure (e.g. wrongly formatted patch, network error, etc.) application 
+will fail. See the [job ConfigMap](../../resources/istio-kyma-patch/templates/configmap.yaml) to learn which patches are 
+applied by default.
 
-1. Unnecessary Istio components are removed - patch looks for file named `delete` which should contain lines in 
-following format:
+4. Remove the unnecessary Istio components. The patch looks for the file named `delete` which should contain lines in 
+the `{kind} {resource-name}` format. Every line describes an Istio resource which should be deleted from the 
+`istio-system` Namespace. It is not an error if the resource to delete is missing. See the 
+[job ConfigMap](../../resources/istio-kyma-patch/templates/configmap.yaml) to see which patches are applied by default.
 
-    ```<kind> <resource-name>```
-    
-    Every line must describe an Istio resource to delete. Resources are deleted from `istio-system` namespace. 
-    It is not an error if resource to delete is missing. See [job ConfigMap](../../resources/istio-kyma-patch/templates/configmap.yaml) 
-    to see which patches are applied by default.
+Patch accepts following environmental variables:
```

Comment:
``The patch accepts the **CONFIG_DIR** environment variable which indicates a directory where patches are placed. If this variable is not set, the script uses the directory where it is placed as default.``